ospf6d/ospf6_top.c

```diff
@@ -911,6 +913,63 @@ DEFUN (no_ospf6_stub_router_admin,
 	return CMD_SUCCESS;
 }
 
+#ifndef VTYSH_EXTRACT_PL
+#include "ospf6d/ospf6_top_clippy.c"
+#endif
+
+/* Ospf Maximum-paths config support */
+DEFPY(ospf6_max_multipath,
+      ospf6_max_multipath_cmd,
+      "maximum-paths (1-64)",
+      "Max no of multiple paths for ECMP support\n"
+      "Number of paths\n")
+{
+	VTY_DECLVAR_CONTEXT(ospf6, ospf6);
+
+	ospf6_maxpath_set(ospf6, maximum_paths);
+
+	return CMD_SUCCESS;
+}
+
+DEFPY(no_ospf6_max_multipath,
+      no_ospf6_max_multipath_cmd,
+      "no maximum-paths",
```

Comment:
You must provide the ability for the operator to specify `no maximum-paths X` when they cut-n-paste their config.

Reply:
Got it. Fixed it.